File: baekjoon/bronze/level2/test_1297/Main.java
package bronze.level2.test_1297;

import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
import java.util.StringTokenizer;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

        StringTokenizer st = new StringTokenizer(br.readLine());
        int D = Integer.parseInt(st.nextToken());
        int H = Integer.parseInt(st.nextToken());
        int W = Integer.parseInt(st.nextToken());

        double k = D / Math.sqrt(H * H + W * W);
        int height = (int) (H * k);
        int width = (int) (W * k);
        System.out.print(height + " " + width);
    }
}

File: baekjoon/bronze/level2/test_1297/kotlin/Main.kt
package baekjoon.bronze.level2.test_1297.kotlin

import kotlin.math.sqrt

fun main() {
    val arr = readln().split(" ")
    val d = arr[0].toInt()
    val h = arr[1].toInt()
    val w = arr[2].toInt()

    val k = d / sqrt((h * h + w * w).toDouble())
    print("${(k * h).toInt()} ${(k * w).toInt()}")
}
